### PR TITLE
Update volunteer recruitment page

### PR DESCRIPTION
**Pull request by ItsiW**
**Web frontend checklist**
- [x] Formatted my code with `yarn format && yarn lint --fix`
- [x] There are no warnings from `yarn lint`
- [x] There are no console warnings when running the app
- [ ] Added any new components to storybook
- [ ] Added tests where relevant
- [ ] All tests pass
- [ ] Clicked around my changes running locally and it works
- [ ] Checked Desktop, Mobile and Tablet screen sizes

### CODE REVIEW

**Review by lucaslcode — CHANGES_REQUESTED** [review]
Just needs to run `yarn format` to pass CI

File: app/web/features/team/Team.tsx [hunk 1]
```diff
@@ -72,43 +72,57 @@ export default function Team() {
...
-                  <div className={classes.cardContent}>
-                    <Typography variant={founder ? "h1" : "h2"} component="h2">
-                      {name}
-                    </Typography>
-                    {founder && (
-                      <Typography variant="h2" component="h3">
-                        Co-founder
+          {TeamData.map(
+            ({ name, director, board_position, role, location, img, link }) => (
+              <Grid key={name} item xs={12} md={!!director ? 4 : 4}>
```
Comment (aapeliv) [inline app/web/features/team/Team.tsx]: Get rid of the `md` conditional

File: app/web/features/team/Team.tsx [hunk 2]
```diff
@@ -72,43 +72,57 @@ export default function Team() {
...
+                <Card
+                  elevation={director ? 3 : 1}
```
Comment (aapeliv) [inline app/web/features/team/Team.tsx:79]: Get rid of the elevation difference and just have their own section for directors, Ireckon

File: app/web/features/team/team.json [hunk 3]
```diff
@@ -277,7 +354,7 @@
   },
   {
     "name": "Marco de Wit",
-    "role": "Frontend Developer",
+    "role": "Web Developer",
```
Comment (aapeliv) [inline app/web/features/team/team.json]: Don't make these changes

File: app/web/markdown/volunteer-documents/guide.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+We want to make sure that you get the most out of your involvement and are able to push Couchers.org forward. We’re sure you’ll meet passionate and skilled people, develop your professional skills, and make this worldwide community even better.
+
+Couchers.org is built by a global team of skilled volunteers — like you — in their spare time. The purpose of this guide is to outline the processes and policies we have in place to make sure we can all do this efficiently and effectively. We want to make sure everything is done at a high standard, while making sure your time spent on it is rewarding and fun.
+
+## Onboarding tasks
+
+To get started, do the following tasks:
+
+1. Get set up on your new google @couchers.org account
+    - make sure you can sign in and view the shared google drive folders
```
Comment (aapeliv) [inline app/web/markdown/volunteer-documents/guide.md]: Big G

File: app/web/redirects.js [hunk 5]
```diff
@@ -44,12 +44,12 @@ module.exports = {
...
       source: "/contributors/guide.pdf",
-      destination: "https://couchers.org/contributors/guide/",
+      destination: "https://couchers.org/volunteer-documents/guide/",
       permanent: false,
     },
     {
-      source: "/contributors/caa.pdf",
-      destination: "https://couchers.org/contributors/caa/",
+      source: "/volunteer-documents/guide.pdf",
+      destination: "https://couchers.org/volunteer-documents/guide/",
```
Comment (aapeliv) [inline app/web/redirects.js]: extra guide?

File: app/web/redirects.js [hunk 6]
```diff
@@ -1,9 +1,9 @@
 module.exports = {
   redirects: [
     {
-      source: "/contributors/form",
+      source: "/volunteer/form",
```
Comment (aapeliv) [inline app/web/redirects.js:4]: wrong url?

Comment (ItsiW) [reply app/web/redirects.js:4]: The form should be associated with the portal from now

**Review by lucaslcode — APPROVED** [review]

**Pull request merged**